implementation/src/main/java/io/smallrye/health/SmallRyeHealthReporter.java

```diff
@@ -71,6 +73,11 @@ private static Throwable getRootCause(Throwable t) {
     public static UncheckedExceptionDataStyle getDefaultUncheckedExceptionDataStyle() {
         return UncheckedExceptionDataStyle.ROOT_CAUSE;
     }
+
+    @Produces
```

Comment:
Does this need the `@Produces`?

Reply:
this seems to be a peculiarity of the CDI environment - it's unhappy without a @Default instance of the enum available

Reply:
That's kind of weird given it's not injected anywhere. @mkouba any thoughts on this?

Reply:
yeah, I thought is was strange as well - ran into it with my previous PR and resolved with this: https://github.com/smallrye/smallrye-health/blob/master/implementation/src/main/java/io/smallrye/health/SmallRyeHealthReporter.java#L70-L73 so just followed suit with this change

Reply:
If it's not injected anywhere then there is no need to turn this into a bean. @emattheis what do you mean _"CDI is unhappy without a default instance"_?

Reply:
@mkouba it _is_ injected by micro profile config into the `emptyChecksOutcome` field. I didn't think it would be necessary to make the enum a bean, but I had to do so with the `UncheckedExceptionDataStyle` enum or else I got a Weld exception at startup, so I followed suit with this change. I can give it a try without the producer later this morning and see what happens...

Reply:
it's actually the TCK that complains at startup: